EmployeeDirectory/README.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Employee Directory
-description: This contacts application is Xamarin.Forms port of the [existing pre-build app](http://xamarin.com/prebuilt/employeedirectory).  With Xamarin.Forms...
+description: 'This contacts application is Xamarin.Forms port of the existing pre-build app. With Xamarin.Forms you can use C or XAML to create layout and this sample app shows how to use both of them. To switch between user interface implementations open ~/EmployeeDirectory/EmployeeDirectoryUI/App.cs file and change uiImplementation variable. Switch to XAML: public static class App { //Change the following line to switch between XAML and C versions private static UIImplementation uiImplementation UIImplementation.Xaml; ... } Switch to C: public static class App { //Change the following line to switch between XAML and C versions private static UIImplementation uiImplementation UIImplementation.CSharp; ... } Nugets used in this solution: [PCLStorage][pclHref], [Xamarin.Forms][xFormsHref], [Microsoft.Bcl][mBclHref], [Microsoft.Bcl.Build][mBclBuildHref], [Microsoft.Net.Http][mNetHttpHref] Components used in this solution: [Xamarin.Social][xSocialHref]'
```

Comment:
For the next iteration, maybe we can limit to the first paragraph so it doesn't pull in too much?